apigateway/views.py

```diff
@@ -66,9 +66,12 @@ def get(self):
 
             if client_id:
                 client, token = extensions.auth_service.load_client(client_id)
+                if token.expires_at() < datetime.now().timestamp():
```

Comment:
There is a function called token.is_expired() that you can use here instead (defined in OAuth2TokenMixin)